api/tests/features/environments/monitors/create.feature

```diff
@@ -11,6 +11,20 @@ Feature: Create Monitor
     When I make a "POST" request to "/environments/e0000001-0000-0000-0000-000000000000/monitors"
     Then the status code should be 403
 
+  Scenario: Checking URL format conforms when creating an environment
+    Given I am logged in
+    When I make a "POST" request to "/environments" with:
+        | field   | value                |
+        | link    | "http://example.com" |
+    Then the status code should be 200
+    And should have a field "link" with "http://example.com"
```

Comment:
This test should go in the `api/tests/features/environments/create.feature` file